packages/manager/src/featureFlags.ts

```diff
@@ -56,6 +56,7 @@ export interface Flags {
   taxCollectionBanner: TaxCollectionBanner;
   databaseBeta: boolean;
   metadata: boolean;
+  premiumPlansAvailabilityNotice: PremiumPlansAvailabilityNotice;
```

Comment:
I'd be okay with just `string` here. 

```suggestion
  premiumPlansAvailabilityNotice: string;
```